core/src/test/scala/org/apache/spark/deploy/SparkHadoopUtilSuite.scala

```diff
@@ -80,6 +82,18 @@ class SparkHadoopUtilSuite extends SparkFunSuite {
     assertConfigValue(hadoopConf, "fs.s3a.endpoint", null)
   }
 
+  /**
+   * test for _HOST pattern replacement with Server cannonical address
+   */
+  test("server principal with _HOST pattern") {
```

Comment:
Maybe let's add a JIRA prefix in the test title: 
```suggestion
  test("SPARK-36622: server principal with _HOST pattern") {
```

Reply:
done